paper-header-panel.html

```diff
@@ -249,22 +278,15 @@
          *       </paper-toolbar>
          *       <div class="content"></div>
          *     </paper-header-panel>
-         *
-         * @attribute mode
-         * @type string
-         * @default ''
          */
         mode: {
           type: String,
-          value: ''
+          value: '',
+          observer: '_modeChanged'
```

Comment:
Should set `reflectToAttribute: true` to reflect to value to attribute so the css you have above ```:host([mode=cover])``` would work.